js/admin/import.js

```diff
@@ -76,6 +76,7 @@ $(function(){
 					$('input[name=skip]').val(jsonData.skip);
 					for (i=0;i<matchs.length;i++)
 						$('#type_value\\['+i+'\\]').val(matchs[i]).attr('selected',true).trigger("chosen:updated");
+					$("select.chosen").trigger("chosen:updated");
```

Comment:
```suggestion
						$('#type_value\\['+i+'\\]').val(matchs[i]).attr('selected',true)
					  $("select.chosen").trigger("chosen:updated");
```

I guess we don't need to update chosen twice

Reply:
we can skip this PR, that old code that was added to dev a month ago solved the issue also. I did not notice it on the dev code.